src/rpc_client/eth_rpc/mod.rs

```diff
@@ -108,7 +108,8 @@ impl ResponseTransform {
 #[candid_method(query)]
 fn cleanup_response(mut args: TransformArgs) -> HttpResponse {
     args.response.headers.clear();
-    ic_cdk::println!(
```

Comment:
I think moving this to `log!` will not work the way you expect, because `cleanup_response` is called as a `query`, meaning that any state modification will be reverted. Same for L. 124

Reply:
The only reason for changing this to `log!` is to use the filtering logic, although I see your point about it not being recorded in the logs due to the query calls. Is there something else I'm missing such that making this change would cause an issue?

Reply:
>  Is there something else I'm missing such that making this change would cause an issue?

I don't see any issue. Maybe add a comment that saying that this log will not be seen in production? Or we could also remove it.

Reply:
I'll remove it for now. Since it's only relevant for local development, we can always add it back again if needed.